utils.groovy

```diff
@@ -199,4 +199,18 @@ def get_push_trigger() {
     ]
 }
 
+def get_artifacts(pipecfg, stream, basearch) {
+    // Get artifacts from config.yaml
+    def artifacts
+    if  (pipecfg.streams."${stream}".artifacts?."${basearch}" != null) {
```

Comment:
Minor: I think another syntax that'd be shorter is: `pipecfg.streams[stream]...`